skydriver/__main__.py

```diff
@@ -6,6 +6,33 @@
 from .config import ENV, LOGGER, config_logging
 
 
+async def database_schema_migration(motor_client) -> None:  # type: ignore[no-untyped-def]
+    from dacite import from_dict
+    from motor.motor_asyncio import AsyncIOMotorCollection
+
+    collection = AsyncIOMotorCollection(
+        motor_client[database.utils._DB_NAME],
+        database.utils._MANIFEST_COLL_NAME,
+    )
+
+    m = 0
+    for i, doc in enumerate([d async for d in collection.find({})]):
+        if "ewms_task" in doc:
+            continue
+        m += 1
+        LOGGER.info(f"migrating {doc}...")
+        doc["ewms_task"] = {
+            "tms_args": doc.pop("tms_args"),
+            "env_vars": doc.pop("env_vars"),
+            "clusters": doc.pop("clusters"),
+            "complete": doc.pop("complete"),
+        }
+        from_dict(database.schema.Manifest, doc)  # validate
+        await collection.find_one_and_update({"_id": doc["_id"]}, doc)
```

Comment:
This probably doesn't do what you think.  You need to use an update verb for the second doc.  Something like: 
```
await collection.find_one_and_update({"_id": doc["_id"]}, {"$set": doc})
```

But maybe `find_one_and_replace()` is a better choice, so it actually deletes the old fields.